hypertension/dc-7101-algorithms/py-root/test/main_test.py

```diff
@@ -1,14 +1,183 @@
+import pytest
 from lib.main import main
 
-def test_main():
-    test_event = {"irrelevant": "value"}
-    assert main(test_event) == {
-        "statusCode": 200,
-        "headers": {
-            "Access-Control-Allow-Headers" : "Content-Type",
-            "Access-Control-Allow-Origin": "*",
-            "Access-Control-Allow-Methods": "OPTIONS,POST"
-        },
-        "body": "test body"
-    }
+@pytest.mark.parametrize(
```

Comment:
These are parametrized test cases - the comment at the top will show what case we're testing (ie `# Both calculator functions return valid results readings`)

Reply:
added this comment in case in was helpful - resolved because closing PR